app/models/health_care_application.rb

```diff
@@ -218,6 +236,30 @@ def parsed_form
     @parsed_form ||= form.present? ? JSON.parse(form) : nil
   end
 
+  def send_event_bus_event(status, next_id = nil)
+    return unless Flipper.enabled?(:hca_kafka_submission_enabled)
+
+    Rails.logger.info '~~~~~~~~~~~~~~~ send_event_bus_event', status, next_id
+
+    begin
+      user_icn = user&.icn || self.class.user_icn(self.class.user_attributes(parsed_form))
+      # local testing
+      # user_icn = user&.icn || '12345678'
+    rescue Common::Exceptions::ValidationErrors
+      # if certain user attributes are missing, we can't get an ICN
+      user_icn = ''
+    end
+
+    Kafka.submit_event(
+      icn: user_icn,
+      current_id: id,
+      submission_name: 'F1010EZ',
+      state: status,
+      next_id:,
+      use_test_topic: true
```

Comment:
You probably were going to do this when taking out the debugger logs but just as a reminder to set use_test_topic to false before merging.

Reply:
Actually, I left it as true on purpose because the real topic is not yet available in staging and I was hoping to do some initial testing with the test topic. We might want to do the same with the 527 form.